casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -356,10 +356,13 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
               //TODO: compare replay deploy cost to given deploy cost
               EvaluateResult(cost, errors) = replayEvaluateResult
               cont <- DeployStatus.fromErrors(errors) match {
-                       case int: InternalErrors => Left((Some(deploy), int)).pure[F]
+                       case int: InternalErrors =>
```

Comment:
What was the compilation error? Was sbt able to successfully compile it?

Reply:
There were many. You can checkout prior to this commit and see for yourself (in IDEA). Obviously yes.